pkg/specgen/generate/container_create.go

```diff
@@ -122,6 +123,19 @@ func MakeContainer(ctx context.Context, rt *libpod.Runtime, s *specgen.SpecGener
 	if err != nil {
 		return nil, nil, nil, err
 	}
+
+	conf, err := rt.GetConfig()
```

Comment:
This is pretty expensive, and liable to undo a lot of the performance improvements @alexlarsson has been making

Reply:
I can use `GetConfigNoCopy()` since i am not editing this so no issues for me.

Reply:
nvm `rtc` variable is already there I don't even need this.

Reply:
removed.